docs/options.md

```diff
@@ -270,6 +270,33 @@ CIBW_BEFORE_BUILD: bash scripts/prepare_for_build.sh
 CIBW_BEFORE_BUILD: "{package}/bin/prepare_for_build.sh"
```

Comment:
This section could use a few tweaks generally. I suggest changing the examples to:

```
# install something required for the build (you might want to use pyproject.toml instead)
CIBW_BEFORE_BUILD: pip install pybind11

# install your package and all dependencies before building (warning: builds your package twice)
CIBW_BEFORE_BUILD: pip install .

# chain commands using &&
CIBW_BEFORE_BUILD_LINUX: yum install -y libffi-dev && make clean

# run a script that's inside your project
CIBW_BEFORE_BUILD: bash scripts/prepare_for_build.sh

# if cibuildwheel is called with a package_dir argument, it's available as {package}
CIBW_BEFORE_BUILD: "{package}/script/prepare_for_build.sh"
```

I've also softened the language a little, to be more appropriate to reference documentation.

Reply:
> install your package and all dependencies before building

This is not true. It still builds before installing. It's just that this build doesn't have a `--no-deps` attached.

```
# build and install your package and all dependencies before building (yes, this builds twice)
```

Would be more accurate, and it highlights the ridiculousness of doing this. :)

Reply:
Haha, well yeah, okay. I get that it's potentially a silly thing to do. We could just remove it from the examples, then? I did a [quick search](https://github.com/search?p=3&q=CIBW_BEFORE_BUILD%3A+pip+install&type=Code) (blimey a lot of people use cibuildwheel now 😄) and it doesn't seem that many projects using `pip install .` anyway.